src/index.html

```diff
@@ -0,0 +1,299 @@
+          <p class="features-section__description">Lorem ipsum dolor sit amet, consetetur sadipscing elitr, sed diam
+            nonumy.</p>
+        </div>
+      </div>
+    </div>
+  </section>
+
+  <section class="solutions-section">
+    <div class="solutions-section__text">
+      <h4 class="solutions-section__label">Why should you work with us?</h4>
+      <h2 class="solutions-section__heading">We have the Best Solution for your Business</h2>
+      <p class="solutions-section__description">Lorem ipsum dolor sit amet,
+        consetetur sadipscing elitr, sed
+        diam nonumy eirmod tempor
+        invidunt ut labore et dolore
+        magna aliquyam erat, sed diam voluptua invidunt ut labore.</p>
+    </div>
+    <div class="solutions-section__list">
+      <div class="solutions-section__list-item">
+        <img src="./assets/icons/red_arrow.svg" alt="arrow icon" class="solutions-section__list-item-icon">
+        <p class="solutions-section__description">Lorem ipsum dolor sit amet, consectetur
+          adipiscing elit, sed do
+          eiusmod.</p>
+      </div>
+      <div class="solutions-section__list-item">
+        <img src="./assets/icons/red_arrow.svg" alt="arrow icon" class="solutions-section__list-item-icon">
+        <p class="solutions-section__description">Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed
+          do
+          eiusmod.</p>
+      </div>
+      <div class="solutions-section__list-item">
+        <img src="./assets/icons/red_arrow.svg" alt="arrow icon" class="solutions-section__list-item-icon">
+        <p class="solutions-section__description">Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed
+          do
+          eiusmod.</p>
+      </div>
+    </div>
+  </section>
+
+  <section class="impressions-section">
+    <div class="impressions-section__text-container">
+      <h2 class="impressions-section__heading">More impressions, more conversions</h2>
+      <p class="impressions-section__description">Lorem ipsum dolor sit amet, consetetur sadipscing elitr, sed diam
+        nonumy eirmod tempor
+        invidunt ut labore et dolore
+        magna aliquyam erat, sed diam voluptua invidunt ut labore.</p>
+      <button type="button" class="impressions-section__button">Get free trial</button>
+    </div>
+    <img class="impressions-section__image" src="./assets/images/Image.png" alt="analytics">
+  </section>
+
+  <section class="logos-section">
+    <img class="logo-section__logo-image" src="./assets/images/Logo 1.png" alt="Logo 1">
+    <img class="logo-section__logo-image" src="./assets/images/Logo 2.png" alt="Logo 2">
+    <img class="logo-section__logo-image" src="./assets/images/Logo 3.png" alt="Logo 3">
+    <img class="logo-section__logo-image" src="./assets/images/Logo 4.png" alt="Logo 4">
+    <img class="logo-section__logo-image" src="./assets/images/Logo 5.png" alt="Logo 5">
+  </section>
```

Comment:
You declared a block `logos-section` but the children are not named after their parent/block. Maybe you meant `logos-section__logo-image`?

Reply:
Yeah I didn't see it 
Thanks

Reply:
You're welcome